ucl-arc/readme.md

```diff
@@ -0,0 +1,28 @@
+Here, and in daughter pages, is the ARC JD framework.
+
+This JD framework defines a generic job description used across all of UCL ARC’s Research Technology Professional Roles. 
+
+Staff are appointed, and roles advertised, with a job title consisting of a seniority prefix and a job family, within this generic JD family.
+
+For staff convenience, individual JDs composed of the appropriate sections from this document may be used as summary
+documents for specific staff or recruitments. 
+
+Each such sub-JD will consist of the preamble, text defining the seniorities appropriate to the grade, 
+and text defining the expertise associated with a job family and specialisation, taken from this document. 
+
+However, the generic JD framework here is the normative document in force.
+
+For each of the  job families, there are a number of possible equivalent titles (separated by a slash “/” below) – staff are able to choose one of these according to preference and roles will be advertised with the option. (To facilitate staff self-identifying in roles that match their career aspirations.)
+Seniority Prefixes: Junior (G6), no-prefix (G7), Senior (G8), Principal (G9) and Professorial (G10)
```

Comment:
We're not using Junior any more! I've updated the wiki source.
(Similarly everywhere in this PR.)

Reply:
I tried to click to merge your changes removing junior, but think I merged the master PR instead. D'oh.